fix: duplicate words (#20195)
mdn/content: files/en-us/web/javascript/reference/statements/with/index.md

}
```

> ### Using `with` with a proxy to create a dynamic namespace

Author: I know you've been striping code sections from headings, but this is one of those cases were you lose context when you do that. I think it is usually a good idea to get rid of the links and bolding, but I don't think code sections should be removed

Reviewer: Not my decision.

I think I prefer:

```suggestion
### Creating dynamic namespaces using the with statement and a proxy
```
